src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.cxx

```diff
@@ -707,7 +707,7 @@ namespace xclhwemhal2 {
               launcherArgs += " -pmc-args-file " + binaryDirectory + "/" + kernels.at(0) + "/emulation_data/pmu_args.txt";
             }
             else {
-              std::cout << "[HW-EMU] ERROR: Unable to find either PMU/PMC args" << std::endl;
+              std::cout << "ERROR: [HW-EMU] Unable to find either PMU/PMC args which are required to launch the emulation." << std::endl;
```

Comment:
Shouldn't emulation use the message infrastructure like the rest of XRT?

Reply:
Sure stsoe, we will take up that as a separate task in next 2020.2 release and make use of XRT infra for messaging even in the hw_emu driver codebase